Description: define PATH_MAX if undefined
 Define PATH_MAX if it is undefined, like in GNU Hurd.
Author: Changwoo Ryu <cwryu@debian.org>
Bug: https://github.com/choehwanjin/libhangul/issues/24
Forwarded: yes

diff --git a/hangul/hangulkeyboard.c b/hangul/hangulkeyboard.c
index 53b9d74..2be6ba4 100644
--- a/hangul/hangulkeyboard.c
+++ b/hangul/hangulkeyboard.c
@@ -24,6 +24,10 @@
 #include <string.h>
 #include <limits.h>
 
+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
+
 #if ENABLE_EXTERNAL_KEYBOARDS
 #include <locale.h>
 #include <glob.h>
